src/docfx/lib/log/Telemetry.cs

```diff
@@ -151,12 +151,24 @@ public static void TrackExternalLink(string tag, string attribute, string scheme
             }
         }
 
-        public static void TrackErrorCount(string code, ErrorLevel level, string? name)
+        public static void TrackErrorCount(Error error)
         {
+            var code = error.Code;
+            var level = error.Level;
+            var name = error.PropertyPath;
+            var additionalInfo = new
+            {
+                error.MsAuthor,
+                error.MsProd,
+                error.MsTechnology,
+                error.MsService,
+                error.MsSubservice,
+            };
+            var additionalInfoString = JsonUtility.Serialize(additionalInfo);
```

Comment:
Consider use `ms.author` or `ms_author` as key in the output JSON. As a guideline, try to use the same name and case for the same term, if `.` is not allowed, replace it with `_`.